server/Context/OfficeDbContext.cs

```diff
@@ -1,133 +1,25 @@
 using Microsoft.EntityFrameworkCore;
+using server.Data;
 using server.Models.Domain;
 
-namespace server.Context
-{
-    public class OfficeDbContext : DbContext
-    {
-        public DbSet<User> Users { get; set; }
-        public DbSet<Seat> Seats { get; set; }
-        public DbSet<Room> Rooms { get; set; }
-        public DbSet<Booking> Bookings { get; set; }
-
-        public OfficeDbContext(DbContextOptions options) : base(options)
-        {
-        }
-
-        protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
This part is very important and should not be removed

Reply:
take back the changed on dbcontext to prod